File: zsbApp.PreviewPdfByFoxit/FoxitPDFViewerForNET200_32914_enu/Examples/Visual Studio 2010_2012_2013_2015/VB/SplitControls/Form1.vb
Public Class Form1
    Public Sub New()
        InitializeComponent()
        ' In the form constructor do not use any Viewer methods (Open, Print etc.) that perform UI activities.
    End Sub
    Private Sub Form1_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        Toolbar1.AssociateToPdfViewer(PdfViewer1)
        NavigationPanel1.AssociateToPdfViewer(PdfViewer1)
        NavigationPanel2.AssociateToPdfViewer(PdfViewer2)
        NavigationPanel2.Hide()
        NavigationPanel2.Dock = DockStyle.None
        NavigationPanel1.Dock = DockStyle.Left
    End Sub

    Private Sub TabControl1_Selected(sender As System.Object, e As System.Windows.Forms.TabControlEventArgs) Handles TabControl1.Selected
        If e.TabPageIndex = 0 Then
            Me.SuspendLayout()
            Toolbar1.AssociateToPdfViewer(PdfViewer1)
            NavigationPanel2.Hide()
            NavigationPanel2.Dock = DockStyle.None
            NavigationPanel1.Dock = DockStyle.Left
            NavigationPanel1.Visible = True
            Me.ResumeLayout()
        Else
            Me.SuspendLayout()
            Toolbar1.AssociateToPdfViewer(PdfViewer2)
            NavigationPanel1.Hide()
            NavigationPanel1.Dock = DockStyle.None
            NavigationPanel2.Dock = DockStyle.Left
            NavigationPanel2.Visible = True
            Me.ResumeLayout()
        End If

    End Sub
End Class

File: zsbApp.PreviewPdfByFoxit/FoxitPDFViewerForNET200_32914_enu/Examples/Visual Studio 2010_2012_2013_2015/CSharp/SplitControls/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace SplitControls
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            // In the form constructor do not use any Viewer methods (Open, Print etc.) that perform UI activities.
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            toolbar1.AssociateToPdfViewer(pdfViewer1);
            navigationPanel1.AssociateToPdfViewer(pdfViewer1);
            navigationPanel2.AssociateToPdfViewer(pdfViewer2);
            navigationPanel2.Hide();
            navigationPanel2.Dock = DockStyle.None;
            navigationPanel1.Dock = DockStyle.Left;
        }

        private void tabControl1_Selected(object sender, TabControlEventArgs e)
        {
            if (e.TabPageIndex == 0)
            {
                this.SuspendLayout();
                toolbar1.AssociateToPdfViewer(pdfViewer1);
                navigationPanel2.Hide();
                navigationPanel2.Dock = DockStyle.None;
                navigationPanel1.Dock = DockStyle.Left;
                navigationPanel1.Visible = true;
                this.ResumeLayout();
            }
            else
            {
                this.SuspendLayout();
                toolbar1.AssociateToPdfViewer(pdfViewer2);
                navigationPanel1.Hide();
                navigationPanel1.Dock = DockStyle.None;
                navigationPanel2.Dock = DockStyle.Left;
                navigationPanel2.Visible = true;
                this.ResumeLayout();
            }

        }
    }
}
